tests/test_funcs.py

```diff
@@ -21,16 +27,18 @@ class TestAsDict(object):
     """
     Tests for `asdict`.
     """
-    def test_shallow(self, C):
+    @given(st.sampled_from([dict, OrderedDict]))
```

Comment:
Let’s have a `DICT_TYPES = (dict, OrderedDict,)`.  Maybe we’ll get more one day and this is quite repetitive.